dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/sequence/FastaSequenceReader.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.dbsnpimporter.sequence;
+
+import htsjdk.samtools.SAMException;
+import htsjdk.samtools.reference.ReferenceSequenceFile;
+import htsjdk.samtools.reference.ReferenceSequenceFileFactory;
+
+import java.io.FileNotFoundException;
+import java.nio.file.Path;
+import java.util.NoSuchElementException;
+
+/**
+ * Implementation of SequenceReader for indexed fasta files
+ */
+public class FastaSequenceReader implements SequenceReader {
+
+    static final String END_LESS_THAN_START_EXCEPTION_MESSAGE = "'end' should be greater or equal than 'start'";
```

Comment:
We don't constants for exception messages anywhere else, let's try to be consistent.

Reply:
I'm just doing this here because I need to check the messages in the tests (not the best practice, but the other option I have is implementing several custom exceptions). I can hardcode the messages in the tests, add the custom exceptions, or leave this at it is.

Reply:
I have created a custom exception and I don't need to check the messages anymore, so I'm deleting those constants.